### DISCUSSION TITLE
How to let nextflow scratch move files in nxf_unstage rather than copy files?

### DISCUSSION THREAD

**Discussion by lishengting** [nextflow-io/nextflow#4773] [Q&A] [answered]
Nextflow scratch copy files (nxf_fs_copy) wasted too much time and disk space in nxf_unstage(). 
Because I set the temporary folder located on the same hard disk with working dir (scratch = '/path/to/work/tmp'), so move or hard link files are more efficcient. 
Is it possible to chage the method from nxf_fs_copy to nxf_fs_move? Thanks!

**Comment by bentsherman** [comment, marked as answer]
Use the `stageOutMode` directive: https://nextflow.io/docs/latest/process.html#stageoutmode

**Reply by lishengting** [reply]
Yes, it worked. Thanks a lot!
